docs/developer/app-store/apps/avatax/configuration.mdx

```diff
@@ -40,7 +34,7 @@ Fields that affect the created transactions:
 
 #### Document recording
 
-Taxes App creates AvaTax transactions on two events:
+AvaTax App creates AvaTax transactions on two events:
```

Comment:
**Question:** should it be `AvaTax` or `Avatax`?

Reply:
I will change to AvaTax everywhere